### ISSUE TITLE
Support showing TM articles mapped to Fortify results when Fortify plugin is NOT installed

### ISSUE THREAD

**Issue by DinisCruz** [TeamMentor/TeamMentor_Eclipse_Plugin#32]
The requirement is to show TM articles that are mapped to Fortify findings, exactly how we do now, but support the scenario when the fortify plugin is not installed (only the TM plugin is installed).

Idea is to load the Fortify findings from the Fortify SSC (Software Security Center), map them to TM articles and show the articles in the UI

Requested by company evaluating TM

**Comment by romichg** [comment]
I don't think that SSC integrates with any IDE. It is basically a dashboard aggregate of all the scans. How would then we integrate with the actual code? i.e. how would we show that the issue is on a particular line of code?

**Comment by DinisCruz** [comment]
The idea is that we (TM IDE Plugin) connect to the SSC server , use it's web services to download findings data (i.e. issues with lines of code) and then show that info in the IDE (mapped to our guidance)

**Comment by romichg** [comment]
I am not even sure SSC provides such ability. Anyhow, we should move this issue to WhatsCooking repo for more discussion.
